### PR TITLE
support margin modifiers in SkeletonView

### PR DESCRIPTION
**Pull request by lidord-wix**
## Description
Support margin modifiers in SkeletonView
WOAUILIB-2175

## Changelog
Support margin modifiers in SkeletonView Component

### CODE REVIEW

File: src/components/skeletonView/index.tsx [hunk 1]
```diff
@@ -407,22 +410,66 @@ class SkeletonView extends Component<SkeletonViewProps, SkeletonState> {
...
-          />
-        );
-      });
+      return (
+        <View {...others}>
+          {_.times(times, index => {
+            const key = timesKey ? `${timesKey}-${index}` : `${index}`;
+            return (
+              <SkeletonView
+                modifiers={{}}
```
Comment (ethanshar) [inline src/components/skeletonView/index.tsx:461]: Can we pass null here instead?

Comment (lidord-wix) [reply src/components/skeletonView/index.tsx:461]: It throws a TS error:
`Type 'null' is not assignable to type 'ExtractedStyle'.ts(2322)
`
I can set it to null and add a `@ts-expect-error`, wdyt?

Comment (ethanshar) [reply src/components/skeletonView/index.tsx:461]: Where the `modifiers` prop typing comes from? 
What about passing `undefined` or is that ignored?

Comment (lidord-wix) [reply src/components/skeletonView/index.tsx:461]: Same error for `undefined` as well.
The type comes from `asBaseComponent`:
` modifiers: ReturnType<typeof Modifiers.generateModifiersStyle>;
`

**Review by ethanshar — APPROVED** [review]

File: src/components/skeletonView/index.tsx (hunk 1 above)
Comment (ethanshar) [reply src/components/skeletonView/index.tsx:461]: Got it, ok. Approving..

**Pull request merged**